portfolio/src/main/webapp/script.js

```diff
@@ -85,13 +86,61 @@ function makeComment(heading, content) {
 function getDropdownVal() {
     const dropdown = document.getElementById("num-comments");
     dropdown.onchange = changeDropdownVal;
-    if (localStorage["num-comments"]) {
-        dropdown.value = localStorage["num-comments"];
+    if (sessionStorage["num-comments"]) {
+        dropdown.value = sessionStorage["num-comments"];
     }   
 }
 
 function changeDropdownVal() {
     const dropdown = document.getElementById("num-comments");
-    localStorage["num-comments"] = dropdown.value;
+    sessionStorage["num-comments"] = dropdown.value;
     this.form.submit();
+}
+
+function deleteMessages() {
+    console.log("Fetching response.");
+    const responsePromise = fetch("/delete-data", {method:"POST"});
+    responsePromise.then(handleDelete);
+}
+
+function handleDelete(response) {
+    console.log("Handling the delete response.");
+    const deletePromise = response.text();
+    deletePromise.then(deleteTextFromDom);
+}
+
+function deleteTextFromDom(text) {
+    console.log("Deleting text from DOM: "+text);
+    const table = document.getElementById("comment-section");
+
+    var i;
+    for (i = 0; i < table.children.length; i++) {
+        const comment = table.children[i];
+        comment.remove();
+    }
+}
+
+function setCurrentPage() {
+    const current = document.getElementById("current-page");
+    if (sessionStorage["current-page"]) {
+        current.innerText = sessionStorage["current-page"];
```

Comment:
`current.innerText = sessionStorage["current-page"]||1` would also work by the way.

try this:
```
a = ''
b = 0
c = 'apple'
d = 57
x = a||b||c||d
console.log(x)
x = a||b||d||c
console.log(x)
x = a||c||b||d
console.log(x)
x = a||d||b||c
console.log(x)
```

Reply:
oh cool i didn't know that, fixed!